src/components/Filters/spec.js

```diff
@@ -192,6 +192,26 @@ describe('Filters component', () => {
           .text()
       ).toBe('Tag is not rarity : 10')
     })
+
+    it('should remove buttons labels if compact property is set to true', () => {
+      const component = mount(
+        <Provider>
+          <Filters compact={[true, true, true, true]} schema={schema} />
```

Comment:
NIT: We could do `true` here (I am thinking we should modify `useBreakpoint` to also accept a single value, and just use `Array.isArray` to check if it is array or not, and if it not, just convert it to one). Might be unnecessary for now though :)

Reply:
I also thought about it and I think it is absolutely to do, so we make the same behaviour of the `StyledSystemProps`. But I thought of doing it in another PR, what do you think? @sradevski

Reply:
@JSReds yeah for sure, since it is an extension of the interface, users won't need to do any changes, so we can do it in a separate PR.